web/app.php

```diff
@@ -27,12 +33,11 @@
 // See http://symfony.com/doc/current/book/http_cache.html#symfony-reverse-proxy
 //
 // require_once __DIR__.'/../app/AppCache.php';
-// $kernel = new AppKernel('prod', false);
-// $kernel->loadClassCache();
 // $kernel = new AppCache($kernel);
 // Request::enableHttpMethodParameterOverride();
```

Comment:
OK then. In the future we'll refactor the application to stop faking the HTTP method with the `_method` hack.

For now I've improved the note about this line as follows:

```
// If you use HTTP Cache and your application relies on the _method request parameter
// to get the intended HTTP method, uncomment this line.
// See http://symfony.com/doc/current/reference/configuration/framework.html#http-method-override
// Request::enableHttpMethodParameterOverride();
```